### PR TITLE
Use App Env from appconfig.json in RUM logs

### PR DESCRIPTION
**Pull request by BenCoffeed**
Shortcut Story ID: [sc-27729]

### CODE REVIEW

File: src/js/datadog.js [hunk 1]
```diff
@@ -2,11 +2,11 @@ import { get } from 'underscore';
 import { datadogRum } from '@datadog/browser-rum';
 import { datadogLogs } from '@datadog/browser-logs';
 
-import { datadogConfig as config, versions } from './config';
+import { datadogConfig as config, versions, appConfig as app} from './config';
 
 function initLogs({ isForm }) {
   datadogLogs.init({
-    env: _DEVELOP_ ? 'develop' : 'prod',
```
Comment (paulfalgout) [inline src/js/datadog.js:9]: We need to add this data to https://github.com/RoundingWell/care-ops-frontend/blob/develop/src/assets/appconfig.json or local is going to fail.  It also removes the necessity for the`app.hasOwnProperty.env` check below

**Review by paulfalgout — CHANGES_REQUESTED** [review]

File: src/js/datadog.js (hunk 1 above)
Comment (BenCoffeed) [reply src/js/datadog.js:9]: Done.

**Comment by paulfalgout** [comment]
@BenCoffeed the errors are legit.  It's a linter issue.  `npm run lint` will likely fix those for you

**Review by paulfalgout — CHANGES_REQUESTED** [review]
linter errors

**Comment by paulfalgout** [comment]
image

**Comment by BenCoffeed** [comment]
Ah man. My bad. On it.

**Pull request merged**